# ==============================================================================
# FIXING COMMIT DIFF
# ==============================================================================
# File: toolkit/components/printing/tests/browser.ini
# Commit: 635806b9356c
# Full Hash: 635806b9356cd535ff079a4cfceb9ca954ac23da
# Author: Yoshi Cheng-Hao Huang <allstars.chh@gmail.com>
# Date: 2022-07-11 15:47:29
# Description:
#   Bug 1777496 - Check if ModuleLoader exists before calling SetAcquiringImportMaps. r=smaug
#   
#   For the print preview documents, they are cloned from the original document in
#   Document::CreateStaticClone[1]. But the cloned document isn't attched to any
#   nsIContentViewer yet so it won't have any global object. The document is
# ==============================================================================

diff -r d83579bec018 -r 635806b9356c toolkit/components/printing/tests/browser.ini
--- a/toolkit/components/printing/tests/browser.ini	Mon Jul 11 09:34:33 2022 +0000
+++ b/toolkit/components/printing/tests/browser.ini	Mon Jul 11 09:38:55 2022 +0000
@@ -66,6 +66,10 @@
   file_coop_header.html
   file_coop_header.html^headers^
 
+[browser_preview_print_link_modulepreload.js]
+support-files =
+  file_link_modulepreload.html
+
 [browser_print_in_container.js]
 skip-if = 
   tsan # Bug 1683730